### PR TITLE
Client side checkout basket

### PR DESCRIPTION
**Pull request by vkaracic**
## New client-side checkout basket documentation.
The feature was introduced in https://github.com/edx/ecommerce/pull/1048
It enables users to fill out the billing information directly in the basket page and pay for their products without the need to go to a 3rd party payment processor website and return back. This is basically how to looks like:
screenshot from 2016-12-21 09-45-59

We need to document a description of it and steps necessary to enable it. I've started with adding a very rough documentation that probably needs polishing, also the waffle flag options should probably be described as well, in a table perhaps? These are the options:
screenshot from 2016-12-21 09-49-17

### Reviewers

- [ ] Subject matter expert: @clintonb 
- [ ] Subject matter expert: @mjfrey 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @vkaracic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fedx-documentation&number=1375"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

File: en_us/install_operations/source/ecommerce/install_ecommerce.rst [hunk 1]
```diff
@@ -315,6 +316,47 @@ instead of ShoppingCart, follow these steps.
...
+* MasterCard
+* Visa
+
+.. note::
+  The payment processor does not save credit card information.
+
+To enable the client-side checkout basket, follow these steps.
+
+#. Sign in to the Django administration console for your base URL. For example,
+   ``http://{your_URL}/admin``.
```
Comment (srpearce) [inline en_us/install_operations/source/ecommerce/install_ecommerce.rst:348]: I added this step based on similar procedures; feel free to remove it if necessary.

File: en_us/install_operations/source/ecommerce/install_ecommerce.rst [hunk 2]
```diff
@@ -315,6 +316,47 @@ instead of ShoppingCart, follow these steps.
...
+
+#. Activate the client side payment processor (eg. CyberSource) switch in the
+   admin panel.
```
Comment (srpearce) [inline en_us/install_operations/source/ecommerce/install_ecommerce.rst:351]: This step could use a few more details—does the user need to access a different page to find this switch? Does she activate it by checking a box, typing something into a field, or doing something else?

File: en_us/install_operations/source/ecommerce/install_ecommerce.rst [hunk 3]
```diff
@@ -315,6 +316,47 @@ instead of ShoppingCart, follow these steps.
...
+
+#. On the **Site Configuration** page, locate the **Client-side payment
+   processor** field, and then enter ``cybersource``.
```
Comment (srpearce) [inline en_us/install_operations/source/ecommerce/install_ecommerce.rst:354]: I made up the "Site Configuration" part—please correct if necessary (include details about how to get to the page where the user sees the "Client-side payment processor" field).

File: en_us/install_operations/source/ecommerce/install_ecommerce.rst [hunk 4]
```diff
@@ -315,6 +316,47 @@ instead of ShoppingCart, follow these steps.
...
+
+#. Edit the **enable_client_side_checkout** waffle flag to the values that you
+   need.
```
Comment (srpearce) [inline en_us/install_operations/source/ecommerce/install_ecommerce.rst:357]: How does the user get to the page where this flag is listed? Does he need to add the flag, or is it already there (so he can just edit it)? Does the user need to set the flag's properties to match all of the fields in the screen shot, or are some of these optional?

**Review by srpearce — COMMENTED** [review]
I added a commit with some changes and included a few questions.

File: en_us/install_operations/source/ecommerce/install_ecommerce.rst (hunk 2 above)
Comment (vkaracic) [reply en_us/install_operations/source/ecommerce/install_ecommerce.rst:351]: It's in Admin > Waffle > Switches.  ``payment_processor_active_cybersource`` needs to be selected and from the dropdown ``Enable the selected switches`` needs to be selected and ``Go`` button needs to clicked.

File: en_us/install_operations/source/ecommerce/install_ecommerce.rst (hunk 3 above)
Comment (vkaracic) [reply en_us/install_operations/source/ecommerce/install_ecommerce.rst:354]: Hmm, perhaps something like:
In your site configurations, ``Admin > Core > Site configurations`` select your site configuration. On that page locate ``Client-side-processor`` field, and enter ``cybersource``. Save the changes.

File: en_us/install_operations/source/ecommerce/install_ecommerce.rst (hunk 4 above)
Comment (vkaracic) [reply en_us/install_operations/source/ecommerce/install_ecommerce.rst:357]: It's located in ``Admin > Waffle > Flags``, it's named ``enable_client_side_checkout`` and it's the only one there. The user changes the field values as needed, each field has a description below it.

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @vkaracic! I've created [OSPR-5325](https://openedx.atlassian.net/browse/OSPR-5325) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Pull request closed without merging**